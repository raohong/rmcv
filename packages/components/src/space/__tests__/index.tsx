import { render, screen } from '@testing-library/react';
import { Space } from '..';

const testId = 'space';

it('render correctly', () => {
  const tree = render(<Space />);

  expect(tree.asFragment()).toMatchSnapshot();
});

it('render with size', () => {
  render(<Space data-testid={testId} size={16} />);

  expect(screen.getByTestId(testId)).toHaveStyleRule('gap', '16px');
});
