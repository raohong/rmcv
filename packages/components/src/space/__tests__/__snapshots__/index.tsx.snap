// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-tap-highlight-color: transparent;
  -webkit-font-smooth: antialiased;
  position: relative;
  touch-action: none;
  list-style: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  font-family: Open Sans,-apple-system,BlinkMacSystemFont,Helvetica Neue,Helvetica,Segoe UI,Arial,Roboto,PingFang SC,miui,Hiragino Sans GB,Microsoft Yahei,sans-serif;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  gap: 8px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
    class="RVuiSpace-root RVuiSpace-horizontal emotion-0"
  />
</DocumentFragment>
`;
